webpack.config.js

```diff
@@ -26,13 +26,17 @@ const config = {
   entry: resolve('./src/index.js'),
   output: {
     path: path.resolve(__dirname, 'dist'),
+    publicPath: '',
     filename: 'demosplan-ui.umd.js',
-    library: '@demos-europe/demosplan-ui',
-    libraryTarget: 'umd',
+    library: {
+      name: '_demos_europe_demosplan_ui',
```

Comment:
I've seen custom library names starting with two underscores instead of one out there. I am however not sure if that is a convention or if I've just been lucky enough to only stumble upon this style.

Reply:
It seems common to use the double underscore start hence adjusted here [f72ac49](https://github.com/demos-europe/demosplan-ui/pull/114/commits/f72ac49ebb2ebf3371ec29df00059f4d21b9d000)